DuckDuckGo/Localizable.strings

```diff
@@ -34,15 +34,17 @@
 "action.title.pasteAndGo" = "Paste & Go";
 "action.title.save" = "Save";
 "action.title.cancel" = "Cancel";
-"action.title.newTab" = "Open in New Tab";
+"action.title.newTab" = "New Tab";
```

Comment:
The strings in Interface Builder use a different system and aren't yet extracted.